test/tests/performance/pbs_equiv_classes_perf.py

```diff
@@ -114,6 +114,10 @@ def test_basic(self):
         self.logger.info('Cycle 1: %d Cycle 2: %d Cycle time difference: %d' %
                          (cycle1_time, cycle2_time, cycle1_time - cycle2_time))
         self.assertGreaterEqual(cycle1_time, cycle2_time)
+        time_diff = cycle1_time - cycle2_time
+        self.perf_test_result(cycle1_time, "different_equi_class", "sec")
+        self.perf_test_result(cycle2_time, "single_equi_class", "sec")
+        self.perf_test_result(time_diff, "time _difference", "sec")
```

Comment:
remove space make it , time_diff_bn_single_diff_equi_classes.

Reply:
Usually we talk about 'equiv' classes, not 'equi' classes